validator/src/main/resources/expected-data-template/eks/aws-sdk-call-metric.mustache

```diff
@@ -123,51 +108,20 @@
       name: Service
       value: {{serviceName}}
     -
-      name: HostedIn.EKS.Cluster
-      value: {{platformInfo}}
-    -
-      name: HostedIn.K8s.Namespace
-      value: {{appNamespace}}
+      name: Environment
+      value: eks:{{platformInfo}}/{{appNamespace}}
     -
       name: RemoteOperation
       value: GetBucketLocation
     -
       name: RemoteService
-      value: AWS.SDK.S3
-    -
-      name: RemoteTarget
-      value: ::s3:::e2e-test-bucket-name-{{testingId}}
-
--
-  metricName: Latency
-  namespace: {{metricNamespace}}
-  dimensions:
-    -
-      name: Service
-      value: {{serviceName}}
-    -
-      name: HostedIn.EKS.Cluster
-      value: {{platformInfo}}
-    -
-      name: HostedIn.K8s.Namespace
-      value: {{appNamespace}}
-    -
-      name: RemoteService
-      value: AWS.SDK.S3
+      value: AWS::S3
     -
-      name: RemoteTarget
-      value: ::s3:::e2e-test-bucket-name-{{testingId}}
-
--
-  metricName: Latency
-  namespace: {{metricNamespace}}
-  dimensions:
+      name: RemoteResourceType
+      value: AWS::S3::Bucket
     -
```

Comment:
Incorrectly removed the following metric aggregation: `[Environment, Service, RemoteService, RemoteResourceIdentifier, RemoteResourceType]`

This happened in both EKS and EC2 for Java